zingolib/src/test_framework/mocks.rs

```diff
@@ -1,5 +1,5 @@
 //! Tools to facilitate mocks for testing
-use zcash_primitives::transaction::TxId;
+//! This file contains mock implementations of lrz types for unit testing with.
```

Comment:
do we want to limit this mod to lrz? I think we want to use this crate for any mock structs right?

Reply:
I see that this PR is suggesting that all internal structs are mocked in their respective files and this mod is for mocking external library structs. I shouldn't have to read the code changes to work this out. A better description / explaination for this PR would give the reviewer context